rustls/Cargo.toml

```diff
@@ -5,7 +5,7 @@ authors = ["Nikolay Kim <fafhrd91@gmail.com>"]
 workspace = "../"
 
 [[bin]]
-name = "server"
+name = "rustls-server"
```

Comment:
Should you update `cargo run` in README?

Reply:
I think it's ok because there's only one binary in this directory

I get some "No such file or directory" panics when running it from the root directory, but then I get those even when running in its own directory with current master.